rtloader/common/builtins/aggregator.c

```diff
@@ -425,15 +425,22 @@ static PyObject *submit_event_platform_event(PyObject *self, PyObject *args)
 
     PyObject *check = NULL;
     char *check_id = NULL;
-    char *raw_event = NULL;
+    char *raw_event_ptr = NULL;
+    Py_ssize_t raw_event_sz = 0;
     char *event_type = NULL;
 
-    if (!PyArg_ParseTuple(args, "Osss", &check, &check_id, &raw_event, &event_type)) {
+    if (!PyArg_ParseTuple(args, "Oss#s", &check, &check_id, &raw_event_ptr, &raw_event_sz, &event_type)) {
```

Comment:
Yes, the change from `s` to `s#` is expected.
Both are parsing a python string. But instead of getting only a `char *` C pointer to the beginning of the string with `s`, we also get the size of the string with `s#`.
The new size parameter is needed in order to build a `[]byte` object with `C.GoBytes(…)` instead of a `string` object with `C.GoString(…)` in [`pkg/collector/python/aggregator.go` here](https://github.com/DataDog/datadog-agent/pull/16084/commits/17b0450a72985c1e6973c496b877744c55b07e18#diff-cb327f15a7856d13ada0f0bb04307dd91907812c5ed17a7a16df5ba232cf6c39R154).